code/Test_definitions/Traffic_Influence.yaml

```diff
@@ -0,0 +1,1030 @@
+    Application (identified by "appId").\
+    \
+    **zone:**
+    The developer can specify in which geographical area he requires the fastest
+    routing toward the nearest Application instance. A "zone" is a smaller
+    geographical area inside a "region". A "zone" is where the Edge Cloud Zone
+    is located.\
+    To add more zones the TI API must be invoked (POST) for each "zone".
+    New "TrafficInfluence" resources are created (with different
+    "trafficInfluenceID"). All the created resources are aggregated by the
+    Application (identified by "appId").\
+    \
+    **appId:**
+    A globally unique identifier associated with the application. This
+    identifier is provided during the application onboarding process.
+    To influence the traffic toward a specific Application. It is the Operator
+    Platform that detects the appropriate Application instance in the selected
+    "region" or "zone".\
+    \
+    **appInstanceId:**
+    A globally unique identifier generated by the Operator Platform to identify
+    a specific instance of the Application on a specific zone. To influence a
+    traffic toward a specific Application instance.\
+    \
+    **sourceTrafficFilters:**
+    The traffic can be from a specific port in the device. If this parameter is
+    used, the influenced flow is from the port defined in "sourceTrafficFilters"
+    rathar than the port specified in "Device"\
+    \
+    **destinationTrafficFilters:**
+    The Application can expose different service on different interfaces,
+    identified by port and protocol, with this parameter it is possible to
+    route the traffic toward a specific port and protocol exposed by the
+    Application.\
+    \
+    **Device:**
+    An user Device can be provided as an input. The Device can be identified by
+    the phone number (phoneNumber), an external identifier
+    (networkAccessIdentifier) or by its IP Address (Ipv4Address, Ipv6Address)
+    also specifying a Port. For IP address both the private (as seen from inside
+    the Device) and the public (as seen over Internet by a server connected to
+    the Device) can be used. To add more users the TI API must be invoked (POST)
+    of each user Device. New "TrafficInfluence" resources are created (with
+    different "trafficInfluenceID"). All the created resources are aggregated by
+    the Application (identified by "appId"). The routing toward the selected
+    Application instance is only applied for provided user Devices. "publicPort"
+    can be used to identify the device. "publicPort" can be also used to
+    identify the flow to be influenced. If the flow to be influenced is from a
+    different port, "sourceTrafficFilters" can be used.\
+    \
+    **Notification URL and token:**
+    Developers have a chance to specify call back URL on which notifications
+    (e.g. session termination) regarding the session can be received from the
```

Comment:
I think so far nowhere it is mentioned the concept of "session" above. So session termination may need to be clarified. May be we should mention in above that the TI API creates a session with a given duration in which events can also be reported to the API consumers and session can be terminated by say DELETE or by the API server too.